src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -155,6 +168,29 @@ public void UsedChangedFor(DataColumnDTO dataColumnDTO, bool used)
       {
          raiseUsedChanged(dataColumnDTO.DataColumn, used);
          updateDataSelection(_view.SelectedColumns);
+         updateLinkedObservedData(dataColumnDTO.DataColumn, used);
+      }
+
+      private void updateLinkedObservedData(DataColumn dataColumn, bool used)
+      {
+         if (!_isLinkedMappedOutputs) return;
+         
+         var linkedObservedData = getLinkedObservedDataFromOutputPath(dataColumn.PathAsString);
+         SetUsedState(linkedObservedData, used);
+      }
+
+      private List<DataColumnDTO> getLinkedObservedDataFromOutputPath(string updatedDataColumnName)
+      {
+         var linkedObservedDataRepositories = AllOutputMappings.AllDataRepositoryMappedTo(updatedDataColumnName);
```

Comment:
Method is called xxxFromOutputPath but the parameter is called `updatedDataColumnName`? Confusing

Reply:
renamed to outputPath